### PR TITLE
Transliteration enabled for sentences

### PR DESCRIPTION
**Pull request by malayaleecoder**
@navaneeth Please have a look. I am not quite sure about any nit convention(Please check).
Just after the confirmation of this, will submit a PR for reverse transliteration also.

Part of #133

### CODE REVIEW

**Comment by navaneeth** [comment]
Found the following issues:

* Output has extra space between words
* `./varnamc -s ml -t malayalam malayalam` is not working. Only the quoted version works.

File: varnamc [hunk 1]
```diff
@@ -978,46 +978,41 @@ def start_compilation
...
-          error error_message
-          exit(1)
-      end
-  end
+  sentence = $options[:text_to_transliterate]
+  sentence = sentence.split(' ')
+  
+  sentence.each do |totl|
+    if $options[:indic_digits]
+        configured = VarnamLibrary.varnam_config($varnam_handle.get_pointer(0),
```
Comment (navaneeth) [inline varnamc]: Why configuration has to be done multiple times? It has to be done only once.

File: varnamc [hunk 2]
```diff
@@ -978,46 +978,41 @@ def start_compilation
...
+    end
 
-  words_ptr = FFI::MemoryPointer.new :pointer
-  done = VarnamLibrary.varnam_transliterate($varnam_handle.get_pointer(0), totl, words_ptr);
-  if done != 0
-    error_message = VarnamLibrary.varnam_get_last_error($varnam_handle.get_pointer(0))
-    puts error_message
-    exit(1)
-  end
+    words_ptr = FFI::MemoryPointer.new :pointer
```
Comment (navaneeth) [inline varnamc]: Please remove leading and trailing whitespaces

File: varnamc [hunk 3]
```diff
@@ -978,46 +978,41 @@ def start_compilation
...
 
-  size = VarnamLibrary.varray_length(words_ptr.get_pointer(0))
-  0.upto(size - 1) do |i|
-    word_ptr = VarnamLibrary.varray_get(words_ptr.get_pointer(0), i)
+    size = VarnamLibrary.varray_length(words_ptr.get_pointer(0))
+    word_ptr = VarnamLibrary.varray_get(words_ptr.get_pointer(0), 0)
     vword = VarnamLibrary::Word.new(word_ptr)
     word = VarnamWord.new(vword[:text], vword[:confidence])
-    puts "  " + word.text
+    print "  " + word.text
```
Comment (navaneeth) [inline varnamc]: Use `puts`

File: varnamc [hunk 4]
```diff
@@ -978,46 +978,41 @@ def start_compilation
...
   end
+  
+  puts "\n"
```
Comment (navaneeth) [inline varnamc]: No need to add an extra line

File: varnamc (hunk 3 above)
Comment (malayaleecoder) [inline varnamc]: @navaneeth `puts` gives the output in different lines. I used print here to display the result in the same line.

File: varnamc (hunk 4 above)
Comment (malayaleecoder) [inline varnamc]: @navaneeth This extra line is needed when using print, else the control will return to the same line.

File: varnamc (hunk 1 above)
Comment (malayaleecoder) [inline varnamc]: Thanks.

File: varnamc (hunk 2 above)
Comment (malayaleecoder) [inline varnamc]: Thanks.

**Comment by malayaleecoder** [comment]
Closing this to create a new one for both transliteration and reverse transliteration.

**Pull request closed without merging**